src/Auth/LogIn/LogIn.test.tsx

```diff
@@ -61,7 +75,7 @@ describe(LogIn, () => {
         </MemoryRouter>,
       )
 
-      const loginButton = await screen.getByRole('button')
+      const loginButton = await screen.getByRole('button', { name: '' })
```

Comment:
why did you add the extra `{ name: '' }` selector here - is it needed?

Reply:
It is, there are two components with the 'button' role and when is still loading the accessible name is an empty string. I tried to give it a meaningful name but failed. I forgot to ask you about it this morning but either way it needs to include name when looking for the component or the test fails.